bittensor/commands/utils.py

```diff
@@ -15,25 +15,69 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-from dataclasses import dataclass
 import os
-import requests
 import sys
+from dataclasses import dataclass
```

Comment:
I understand almost all of these import order changes, but I don't get this one. This is non-blocking. Just looking for the rationale.

Reply:
There is a convention for the sequence of correct import of packages: first those built into Python-SDK, then installed (third party) and then everything that is implemented locally (in the product being developed). This is mentioned in the PEP.

Reply:
Correct. But `dataclasses` is a stdlib package.

Reply:
Indeed, if I did `import dataclasses`, then use it in the code as `@ dataclasses. dataclass`
But in every section I mentioned, `import %package%` is done first, and then everything that is `from %package% import %object%`

Reply:
Okay. I've never heard of that style of importing. I've always just used the PEP8 convention of stdlib,pypi,local (sorted alphabetically).